src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Button/Button_Events.xaml.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Windows.UI.Xaml.Controls;
+using Uno.UI.Samples.Controls;
+
+namespace UITests.Shared.Windows_UI_Xaml_Controls.Button
+{
+	[SampleControlInfo("Button", "Button_Events")]
+	public sealed partial class Button_Events : Page
+	{
+		private int _btnTappedCount;
+		private int _btnDoubleTappedCount;
+		private int _btnClickCount;
+		private int _btnPointerPressedCount;
+		private int _btnPointerReleasedCount;
+		private int _btnPointerEnteredCount;
+		private int _btnPointerExitedCount;
+
+		public Button_Events()
+		{
+			this.InitializeComponent();
+
+			btnTapped.Tapped += (snd, evt) => btnTappedCount.Text = (++_btnTappedCount).ToString();
```

Comment:
No UI Tests for that sample ?

Reply:
I made some UI tests for `Button` pointer events, I will merge them once @dr1rrb 's feature branch is merged.

Reply:
Yes there's a pretty big one.